block_model.rb

```diff
@@ -0,0 +1,73 @@
+class BlockModel
+  attr_reader :blocks
+
+  def initialize(blocks)
+    @blocks = blocks
+  end
+
+  def reblock(rx, ry, rz)
```

Comment:
Just before line 8 there should be a commentary explaining what is the functionality of the reblock function, what are the parameters needed ( and what  are theese parameters for ) and what does this function return.